build.sbt

```diff
@@ -80,6 +81,9 @@ lazy val azureStorageQueue = alpakkaProject("azure-storage-queue", "azure.storag
 
 lazy val cassandra = alpakkaProject("cassandra", "cassandra", Dependencies.Cassandra)
 
+lazy val couchbase =
+  alpakkaProject("couchbase", "couchbase", Dependencies.Couchbase, parallelExecution in Test := false)
```

Comment:
Using parallel execution is problematic here because  I run FlowSpec in the same Couchbase bucket on the same dataset. So running updatebulk and deletebulk tests simultaneously would cause the problem, due to the Couchbase Optimistic locking nature. In my opinion, creating every time new dataset with different ids for 19 different unit tests will produce a lot of code bloat.  all couchbase test suite runs for 17 seconds form which about 15 seconds take to initiate full couchbase cluster with buckets and indexing services and  which is required for running  "Source from query" specs.